pkgs/by-name/sy/syzkaller/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  go,
+  ncurses,
+}:
+let
+  rev = "67d7ec0a69193b2acbb8cdb5109d5505a9128ac5";
+  commitDate = "2024-01-09";
+  version = "unstable-${commitDate}";
+
+  dateString = lib.strings.concatStringsSep "-" [
+    (lib.strings.replaceChars [ "-" ] [ "" ] commitDate)
```

`lib.replaceChars` is a deprecated alias of [lib.replaceStrings](https://noogle.dev/f/lib/replaceStrings.html).

```suggestion
    (lib.strings.replaceStrings [ "-" ] [ "" ] commitDate)
```